project/web/balancer.sls

```diff
@@ -164,31 +164,20 @@ nginx_conf:
 {% if letsencrypt %}
 # Now that we have nginx running, we can get a real certificate.
 
-# To install letsencrypt for now, just clone the latest version and invoke the
-# ``letsencrypt-auto`` script from there. There's not a packaged version of
-# letsencrypt for Ubuntu yet, but once there is, we should switch to that.
-really_reset_letsencrypt:
-  cmd.run:
-    - name: cd {{ letsencrypt_dir}} && git reset --hard HEAD
-    - onlyif: test -e {{ letsencrypt_dir }}/.git
-
-install_letsencrypt:
-  git.latest:
-    - name: https://github.com/letsencrypt/letsencrypt/
-    - target: {{ letsencrypt_dir }}
-    - force_reset: True
-    - require:
-        - cmd: really_reset_letsencrypt
+install_certbot:
+  file.managed:
+    - name: /usr/local/bin/certbot-auto
+    - source: https://dl.eff.org/certbot-auto
+    - source_hash: sha256=fcff7a6a899d1ab576369e5849087b3db7e6cf251e1ea2c963e11b05dfe62e5c
```

Comment:
I suspect they'll be updating the version of certbot-auto at that link from time to time, and this hash check will then fail. We should probably just skip it.